Makefile

```diff
@@ -565,15 +565,17 @@ cross-rpm.%: docker/cross-build/% clean-spec spec dist
 	$(Q)distro=$(patsubst cross-rpm.%,%,$@); \
 	builddir=$(BUILD_DIR)/docker/$$distro; \
 	outdir=$(PACKAGES_DIR)/$$distro; \
+	user=`id -un`; \
 	echo "Docker cross-building $$distro packages..."; \
 	mkdir -p $(PACKAGES_DIR)/$$distro && \
 	rm -fr $$builddir && mkdir -p $$builddir/{input,build} && \
 	cp cri-resource-manager-$(TAR_VERSION).tar$(GZEXT) $$builddir/input && \
 	cp packaging/rpm/cri-resource-manager.spec $$builddir/input && \
-	$(DOCKER) run --rm -ti $(DOCKER_OPTIONS) --user $(shell echo $$USER) \
+	$(DOCKER) run --rm -ti $(DOCKER_OPTIONS) --user $$user \
```

Comment:
Out of curiosity: is setting $user then dereferencing it vs. using $USER a workaround/fix for running in non-login shells ? IOW, what does it fix ?

Reply:
Changed it to `$USER` to keep the changes minimal (and simplify a bit)